openprescribing/frontend/views/views.py

```diff
@@ -311,32 +311,20 @@ def measure_for_all_ccgs(request, measure):
     return render(request, 'measure_for_all_ccgs.html', context)
 
 
-def measure_for_one_practice(request, measure, practice_code):
-    practice = get_object_or_404(Practice, code=practice_code)
+def measure_for_one_entity(request, measure, entity_code, entity_type=None):
```

Comment:
Why would `entity_type` ever be `None`?

Reply:
Ah yes, good spot. I copied that without thinking from a function which handles the "All England" case.